### PR TITLE
feat: Adds load balancer to demonstrate horizontal scaling

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

**Review by sandrask — APPROVED** [review]

File: test/bdd/features/did-orb.feature [hunk 1]
```diff
@@ -123,41 +123,41 @@ Feature:
...
+     Then container "orb-domain1.backend" is started
+     Then container "orb2-domain1.backend" is started
      Then container "orb-domain2" is started
      Then container "ipfs" is started
 
      Then we wait 5 seconds
 
      # now that servers re-started we should check for DIDs that were created before shutdowns
-     Then client sends request to "https://orb.domain1.com/sidetree/v1/identifiers,https://orb2.domain1.com/sidetree/v1/identifiers,https://orb.domain2.com/sidetree/v1/identifiers" to verify the DID documents that were created
+     Then client sends request to "https://orb.domain1.com/sidetree/v1/identifiers,https://orb.domain1.com/sidetree/v1/identifiers,https://orb.domain2.com/sidetree/v1/identifiers" to verify the DID documents that were created
```
Comment (bstasyszyn) [inline test/bdd/features/did-orb.feature:154]: This has two orb.domain1.com.

File: test/bdd/features/did-orb.feature [hunk 2]
```diff
@@ -123,41 +123,41 @@ Feature:
...
 
      # write batch of DIDs to multiple servers again and check them
-     When client sends request to "https://orb.domain1.com/sidetree/v1/operations,https://orb2.domain1.com/sidetree/v1/operations,https://orb.domain2.com/sidetree/v1/operations" to create 50 DID documents using 10 concurrent requests
+     When client sends request to "https://orb.domain1.com/sidetree/v1/operations,https://orb.domain1.com/sidetree/v1/operations,https://orb.domain2.com/sidetree/v1/operations" to create 50 DID documents using 10 concurrent requests
```
Comment (bstasyszyn) [inline test/bdd/features/did-orb.feature:157]: This has two orb.domain1.com.

File: test/bdd/features/did-orb.feature [hunk 3]
```diff
@@ -123,41 +123,41 @@ Feature:
...
 
      Then we wait 5 seconds
-     Then client sends request to "https://orb.domain1.com/sidetree/v1/identifiers,https://orb2.domain1.com/sidetree/v1/identifiers,https://orb.domain2.com/sidetree/v1/identifiers" to verify the DID documents that were created
+     Then client sends request to "https://orb.domain1.com/sidetree/v1/identifiers,https://orb.domain1.com/sidetree/v1/identifiers,https://orb.domain2.com/sidetree/v1/identifiers" to verify the DID documents that were created
```
Comment (bstasyszyn) [inline test/bdd/features/did-orb.feature:160]: This has two orb.domain1.com.

File: test/bdd/features/did-orb.feature (hunk 1 above)
Comment (fqutishat) [reply test/bdd/features/did-orb.feature:154]: we have load balancer

Comment (fqutishat) [reply test/bdd/features/did-orb.feature]: done

File: test/bdd/features/did-orb.feature (hunk 2 above)
Comment (fqutishat) [reply test/bdd/features/did-orb.feature]: done

File: test/bdd/features/did-orb.feature (hunk 3 above)
Comment (fqutishat) [reply test/bdd/features/did-orb.feature]: done

**Review by bstasyszyn — APPROVED** [review]

**Pull request merged**